src/npcs/quests/alienquest.lua

```diff
@@ -74,6 +74,15 @@ local quests = {
     },
     successPrompt = "Could you deposit this document into the mailbox? And no montages!",
     completeQuestFail = "What the hell are you still doing here? Go hold those aliens off!",
+    completeQuestSucceed = {
+      "You...you've done it! You've defeated the {{orange}}QFO{{white}}! I can't believe it! Now I can eat Mexican food in peace, forever!",
+      "What, the aliens didn't automatically die when the mothership blew up? Well, I guess that's pretty realistic, I don't know what I was thinking.",
+      "I know I've treated you unfairly human, but you have my gratitude.",
+      "As a token of thanks, I'll give you my standard issue {{blue_light}}alien pistol{{white}}, you'll need it more than I do. Here's some gold as well.",
+      "Whenever you run out of ammo for the pistol, come back to me, I will sell some to you.",
+      "It was nice working with you partner. We've defeated them!",
```

Comment:
Same deal with this text:
"You...you've done it! You've defeated the {{orange}}QFO{{white}}!  Now I can eat Mexican food in peace, forever!",
"As a token of thanks, you can have my standard issue {{blue_light}}alien pistol{{white}}, you'll need it more than I do. Here's some gold as well.",
"Whenever you run out of ammo for the pistol come back to me and I will sell some to you.",
"It was nice working with you partner. We've defeated them!",